src/main/kotlin/me/clip/voteparty/VoteParty.kt

```diff
@@ -194,13 +210,13 @@ class VoteParty internal constructor(internal val plugin: VotePartyPlugin) : Sta
 	
 	private fun loadVotes()
 	{
-		votesHandler.setVotes(conf().getProperty(PluginSettings.COUNTER))
+		votesHandler.setVotes(voteData().getProperty(VoteData.COUNTER))
 	}
 	
 	private fun saveVotes()
 	{
-		conf().setProperty(PluginSettings.COUNTER, votesHandler.getVotes())
-		conf().save()
+		voteData().setProperty(VoteData.COUNTER, votesHandler.getVotes())
+		voteData().save()
```

Comment:
I'm fairly certain this is being called in sync - see line 91. Thats a no-no for file io.

This rapid fire saving (once per second?!) isn't advisable as it would wear out flash devices. Not quickly, but this feels excessive.